### DISCUSSION TITLE
Memory limits

### DISCUSSION THREAD

**Discussion by tonyakiki** [Igalia/cog#556] [Q&A]
It seems that cog use as much memory as it is available which can lead to other problem when running with other applications on an embedded devices. My question is how to limit the memory using **--web-mem-limit** and **--net-mem-limit**? what is the difference between them and how do they affect each other?
I tried to configure them but i cannot see that they are taking effect... Is there any detailed documentation on that and/or recommendations? Could someone provide an example on how it is supposed to be configured and how to check that it took effect?

**Comment by gnulux** [comment]
Hi tonyakiki 

Did you find an answer to your request ? 
I'm wondering if there is a way to limit WebProcess task in term of memory or if there is a garbage collector which can be set.

Rgds
